.github/workflows/plugin_test.yaml

```diff
@@ -28,13 +28,13 @@ jobs:
             branch: main
             tests_to_run: tests/.
           - plugin: pynxtools-mpes
-            branch: main
+            branch: update-to-new-nxmpes
             tests_to_run: tests/.
           - plugin: pynxtools-stm
             branch: main
             tests_to_run: tests/.
           - plugin: pynxtools-xps
-            branch: main
+            branch: update-configs-after-mpes-changes
```

Comment:
This needs to be updated before merging as well

Reply:
done